dev/com.ibm.ws.anno/resources/com/ibm/ws/anno/resources/internal/AnnoMessages.nlsprops

```diff
@@ -326,23 +326,23 @@ ANNO_CLASSINFO_INTERFACE_MISSORDERED_ADD.useraction=CMSG0002
 
 ANNO_TARGETS_SCAN_EXCEPTION=CWWKC0044W: An exception occurred while scanning class and annotation data. The exception was {0}.
 ANNO_TARGETS_SCAN_EXCEPTION.explanation=An error occurred while scanning class and annotations data.  The associated module may function incorrectly.
-ANNO_TARGETS_SCAN_EXCEPTION.useraction=See the exception and the associated FFDC message for more information.
+ANNO_TARGETS_SCAN_EXCEPTION.useraction=For more information on the topic, see the exception and the associated FFDC message.
```

Comment:
Change
'For more information on the topic, see the exception and the associated FFDC message.'
to
'For more information, see the exception and the associated FFDC message.'

Reply:
Updated.